w3-aggregation.md

```diff
@@ -127,20 +127,23 @@ type AggregateOffer struct {
 
 type AggregateOfferDetail struct {
   offer OfferCBOR
-  commitmentProof Proof
+  segment Proof
```

Comment:
What is the difference betrween segment and piece?

Reply:
segment is commD (aggregate), in other words it is computed by a tree of commP (pieces)